build-tools/packages/build-cli/biome.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The other packages seem to be missing this file

Reply:
Ahhh, this is a leftover from when I was testing organized imports. I'll remove it.